canonical-web-and-design/snapcraft.io
# Redirect account details

## Done
Added a redirect to the new account details page and removed old view

## How to QA
- Go to https://snapcraft-io-4588.demos.haus/account/details
- Check that you are redirected to https://snapcraft-io-4588.demos.haus/admin/account

## Issue / Card
Fixes https://warthogs.atlassian.net/browse/WD-9994
